server/models/k8s_context.go

```diff
@@ -163,15 +163,25 @@ func K8sContextsFromKubeconfig(kubeconfig []byte, instanceID *uuid.UUID) ([]*K8s
 			respMessage += msg
 			continue
 		}
+		
+		// Perform Ping test on the cluster
+		if err := kc.PingTest(); err != nil {
+			msg = fmt.Sprintf("Skipping context: %v \n", err)
+			logrus.Warn(msg)
```

Comment:
Don't use logrus.

Reply:
It's good that a skipping content message is sent. Where is the explanation as to why the context is being skipped?

Reply:
It's good that a skipping content message is sent. Where is the explanation as to why the context is being skipped?

Reply:
The error returned is also added in the msg which contains the reasonl.

Reply:
"Skipping context: Kube API not reachable." or something more accurate than reachable.... ?